### ISSUE TITLE
About pixcontrast loss

### ISSUE THREAD

**Issue by jsjy1** [zdaxie/PixPro#29]
Thank you very much for your work. After reading the code, I have a question.
In the paper, the final loss consists of $l_{pixcontrast}, l_{pixpro}  and  l_{instance}$.
But In the Pixpro.py, I just find the $l_{pixpro}$:
```python
        # compute loss
        loss = regression_loss(pred_1, proj_2_ng, coord1, coord2, self.pixpro_pos_ratio) \
            + regression_loss(pred_2, proj_1_ng, coord2, coord1, self.pixpro_pos_ratio)
```
and $l_{instance}$:
```python
      loss_instance = self.regression_loss(pred_instance_1, proj_instance_2_ng) + \
                   self.regression_loss(pred_instance_2, proj_instance_1_ng)
```
So I wonder where $l_{pixcontrast}$ is, do I understand wrong or where to see the negligence?

**Comment by jsjy1** [comment]
It seems that I misunderstood when I read the paper.
